### ISSUE TITLE
Vue.Js 2 with Typescript is not allowing to import this package

### ISSUE THREAD

**Issue by prindacerk** [s4l1h/vue-toastr#30]
I am trying to import vue-toastr in boot.ts. I have installed the package using npm. It installed successfully and I can see the files.

But when I am trying to import using the following code, it is not working.
`import Toastr from "vue-toastr";`
The error message I am getting is the following.
> TS7016: Could not find a declaration file for module 'vue-toastr'. 'D:/Core.Vue/node_modules/vue-toastr/dist/vue-toastr.min.js' implicitly has an 'any' type.

Do you have any idea what is causing this issue and how to resolve it?

**Comment by s4l1h** [comment]
can you try this.
import * as Toastr from 'vue-toastr';

I dont know typescript how to import working.

Maybe you have to change js compiler.Are you using with babel compiler?
http://www.jbrantly.com/es6-modules-with-typescript-and-webpack/
// with babel compiler very easy.
import Toastr from 'vue-toastr'
Vue.use(Toastr)

**Comment by prindacerk** [comment]
Hi,

I got around that issue by using this code.
`var toastr = require("vue-toastr");`

And then I configured it as plugin.
`Vue.use(toastr);`

But when I try to use the Toastr in the component,
`console.log(this.$toastr);`
 it throws as an error.
![image](https://user-images.githubusercontent.com/4536573/36516356-a84f0738-17d1-11e8-99d4-c557321484ef.png)

I am using Vue.js 2.5.13 and Typescript 2.7.2. I think the plugin build is different for 2.5 correct? I am new to Vue.js so I may be wrong.

Please kindly help as I want to learn Vue.js and right now I am at a loss.

**Comment by s4l1h** [comment]
@prindacerk sorry but i dont have information about typescipt.
You can use very easy with babel.Look at this code I wrote for you.
https://codesandbox.io/s/326no72v6q

**Comment by prindacerk** [comment]
@s4l1h Thanks. I was able to get your plugin working on a Vue.js project I built using js instead of Typescript. The injection coding style is different between both it seems. Your component works fine on js version but not on Typescript version.

However, I was able to get CodeSeven's Toastr imported and working on Typescript. So I was able to use that in my Typescript project.

Anyways, thanks for the help. I was hoping your component would work with Typescript but unfortunately it didn't. Appreciate the support.
